skogul.service

```diff
@@ -6,7 +6,7 @@ Documentation=man:skogul(1) https://github.com/telenornms/skogul
 After=network-online.target
 
 [Service]
-ExecStart=/usr/local/bin/skogul -f /etc/skogul/default.json
+ExecStart=/usr/bin/skogul -show -f /etc/skogul/default.json
```

Comment:
This.... was a bit too quick :D 

Now the path is correct, but instead of having ExecStartPre=... -show and ExecStart=, we only have -show, so it will never start.